dynamallow/model.py

```diff
@@ -98,7 +91,11 @@ class MarshModel(object):
     more information.
 
     The Marshmallow Schema is defined in a class named ``Schema``, which should be filled out exactly as you would fill
-    out any other :class:`~marshmallow.Schema`.
+    out any other :class:`~marshmallow.Schema` or :class:`~schematics.Schema`. The only caveat is that
```

Comment:
For the class link to `schematics` to work correctly you need to add an intersphinx link inside of `docs/conf.py` (assuming they have sphinx docs).